pwndbg/gdblib/config.py

```diff
@@ -156,6 +158,6 @@ def _value_to_gdb_native(value, param_class=None):
 
 def init_params() -> None:
     # Create a gdb.Parameter for each parameter
-    for p in pwndbg.gdblib.config.params.values():
+    for p in config.params.values():
```

Comment:
Is this correct? I believe it may not be - if it weren't correct, we would have caught it already

Reply:
this is correct because you are using config from the same file, and no need to use the whole package path here. Also it worked because of moving the object refs to global scope of the package. Let me revert it.

Reply:
reverted and fixed!